app/models/order.rb

```diff
@@ -16,19 +16,34 @@ def total
   end
 
   def display_status
-    status.capitalize
+    status.capitalize if status
```

Comment:
@TheKaterTot I think this should be reverted back since status should never be nil.